File: C/carrot_juice.c
#include <stdio.h>
#include "growing.h"
#include "pressing.h"
int main()
{
  int field[1000], carrots[1000];
  float juice;
  int i;
  
  plant(field, 1000, 1);
  grow(field, 1000, 3);
  harvest(field, carrots, 1000);
  printf("carrots[3]=%d\n", carrots[3]);
  juice = press(carrots, 1000, 3.14);

  printf("%f carrot juices successfully created\n", juice);
  return 0;
}

File: carrot_juice.f90
PROGRAM carrot_juice

  use growing
  use pressing

  implicit none

  integer, dimension(1000) :: field, carrots
  real :: juice

  
  call plant(field, 1000, 1)
  call grow(field, 1000, 3)
  call harvest(field, carrots, 1000)
  call press(carrots, 1000, 3.14, juice)

  print*, juice, " carrot juices successfully created"
END PROGRAM !carrot_juice
